core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/Repository.java

```diff
@@ -56,4 +56,15 @@
 	 */
 	void sendNotification(RepositoryEvent event);
 
+	/**
+	 * @return Returns the {@link Health health} state for this repository. The repository is considered healthy if it is in {@link Health#GREEN} state.
```

Comment:
Please remove the `Returns` word from `@return` part of the javadoc (this way it reads like `Returns: Returns...`).